### DISCUSSION TITLE
Duplicate getters and setters

### DISCUSSION THREAD

**Discussion by maciej-laskowski** [cvergne/vscode-php-getters-setters#29] [Ideas]
Hi, thank you for maintaining this extension. Have you considered checking for duplicates before inserting getters and setters and then inserting only those that are missing?

**Comment by cvergne** [comment]
Hi, no but that's a good idea :)
I'll take a look when I'll have time for, but if you or anyone want to submit a PR about that, feel free to do it !

**Reply by ctf0** [reply]
i will handle this
